### PR TITLE
Update dependencies and fix compilation errors

### PR DESCRIPTION
**Pull request by vmarkushin**

### CODE REVIEW

File: algorithms/beefy/prover/src/lib.rs [hunk 1]
```diff
@@ -307,7 +307,7 @@ where
 			signed_commitment.commitment.block_number.into();
 		let block_hash = self.relay_client.rpc().block_hash(Some(subxt_block_number)).await?;
 
-		let current_authorities = api.storage().beefy().authorities(block_hash).await?.0;
+		let current_authorities = api.storage().beefy().authorities(block_hash).await?;
```
Comment (seunlanlege) [inline algorithms/beefy/prover/src/lib.rs:310]: This isn't compatible with polkadot v0.9.27

File: light-clients/ics07-tendermint/src/client_state.rs [hunk 2]
```diff
@@ -143,7 +143,7 @@ impl<H> ClientState<H> {
 	}
 
 	pub fn client_type() -> ClientType {
-		"07-tendermint".to_string()
+		"07-tendermint".to_owned()
```
Comment (seunlanlege) [inline light-clients/ics07-tendermint/src/client_state.rs:146]: alloc::borrow::ToOwned is not in scope. Try compiling for wasm target

**Review by seunlanlege — CHANGES_REQUESTED** [review]
Seems to have reverted updates for polkadot-v0.9.27 and not checked for wasm target.

File: algorithms/beefy/prover/src/lib.rs (hunk 1 above)
Comment (vmarkushin) [reply algorithms/beefy/prover/src/lib.rs:310]: It's strange, because this code wasn't compiling

**Pull request merged**